src/scripts/main.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+const links = Array.from(document.querySelectorAll('.list-item__link'));
+const mainImg = document.querySelector('#largeImg');
+
+links.forEach(link => {
+  link.addEventListener('click', (events) => {
+    mainImg.src = link.href;
+    events.preventDefault();
+  });
```

Comment:
```suggestion
  link.addEventListener('click', (event) => {
    mainImg.src = link.href;
    event.preventDefault();
  });
```

Reply:
there is one event :)

Reply:
@sTorba24 It writes an error "'event' is already declared in the upper scope", so I changed it to events